<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlItem</class>
 <widget class="QWidget" name="ControlItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="rhoLabel">
       <property name="text">
        <string>rho</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="rhoDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="thetaLabel">
       <property name="text">
        <string>theta</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="thetaDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>0.015000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="thresholdLabel">
       <property name="text">
        <string>threshold</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="thresholdSpinBox">
       <property name="minimum">
        <number>-9999</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>85</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="srnLabel">
       <property name="text">
        <string>srn</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="srnDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stnLabel">
       <property name="text">
        <string>stn</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="stnDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="min_thetaLabel">
       <property name="text">
        <string>min_theta</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="min_thetaDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.002000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="max_thetaLabel">
       <property name="text">
        <string>max_theta</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="max_thetaDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.002000000000000</double>
       </property>
       <property name="value">
        <double>3.141594000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="select_image_button">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="do_button">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rhoDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thetaDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>srnDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stnDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_thetaDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_thetaDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
